src/library/scala/Predef.scala

```diff
@@ -616,7 +616,7 @@ object Predef extends LowPriorityImplicits {
   /** @group type-constraints */
   object <:< {
     /** If `A <: B` and `B <: A`, then `A = B` (subtyping is antisymmetric) */
-    def antisymm[A, B](implicit l: A <:< B, r: B <:< A): A =:= B = =:=.singleton.asInstanceOf[A =:= B]
+    def antisymm[A, B](implicit @deprecated("unused","shim") l: A <:< B, @deprecated("unused","shim") r: B <:< A): A =:= B = =:=.singleton.asInstanceOf[A =:= B]
```

Comment:
I'd avoid doing this, I find looking at this much more annoying than the lint warning (which we never see). Especially in the standard library, but even in the compiler.

Reply:
Yes, I'll rebase over the PR which doesn't warn. Because I see the warnings. At night in my dreams.

Reply:
Actually, this was at the beginning of the edits, when I thought I wanted to minimize the compiler output, but by the end it wasn't so bad, with deprecations turned off of course.